### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by AlvinSchiller**
Updates for issue templates

- Use new "issue form templates" to enforce giving crucial information
- added version labels to templates
- removed v2 feature request form
- updated links in Readme
- updated icons in Readme to be consistent

Can be viewed in [my fork ("develop") ](https://github.com/AlvinSchiller/RPi-Jukebox-RFID/issues/new/choose)

## Overview
issues_template_overview
## V2 Bug
issues_template_bugV2
## V3 Bug
issues_template_bugV3
## V3 Feature
issues_template_featureV3

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_v3.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      required: true
+
+  - type: input
+    id: branch
+    attributes:
+      label: Branch
+      description: |
+        What branch did you install from?
+        See the "settings" page in the Web App or the "src/jukebox/jukebox/version.py" file.
+      placeholder: e.g. "main" or "develop"
```
Comment (s-martin) [inline .github/ISSUE_TEMPLATE/bug_v3.yaml:30]:
```suggestion
      placeholder: e.g. "future3/main" or "future3/develop"
```

File: .github/ISSUE_TEMPLATE/bug_v3.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        What you did, what you expected and what happened instead?
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        Please provide the logs after the problem occured.
```
Comment (s-martin) [inline .github/ISSUE_TEMPLATE/bug_v3.yaml:77]: Add web interface at `http://ip.of.your.box/logs`  as well. Could be easier for some users.

**Review by s-martin — APPROVED** [review]

**Comment by AlvinSchiller** [comment]
Good to go?

**Comment by s-martin** [comment]
Yes

**Pull request merged**